packages/config/config/devices/0x018c/pm-b400zw-n.json

```diff
@@ -0,0 +1,71 @@
+{
+	"manufacturer": "Dawon DNS",
+	"manufacturerId": "0x018c",
+	"label": "PM-B400ZW-N",
+	"description": "Smart Plug with Power Meter",
+	"devices": [
+		{
+			"productType": "0x0042",
+			"productId": "0x0006",
+			"zwaveAllianceId": 1870
+		},
+		{
+			"productType": "0x0042",
+			"productId": "0x0005"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1[0xff]",
```

Comment:
Technically it doesn't matter whether this is a 0x01 or 0xff, but this way it doesn't look like there may be another function hidden in the other 7 bits

Reply:
Also it made more sense to have the enable/disable before the value setting